app/src/main/res/layout/dialog_error.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/errorDialog"
+    android:layout_width="300dp"
+    android:layout_height="wrap_content"
+    android:background="@drawable/dialog_background"
+    android:orientation="vertical">
+
+
+    <TextView
+        android:id="@+id/title"
+        fontPath="fonts/Roboto-Regular.ttf"
```

Comment:
Note : I find it easy to keep the font path in a string and use @string/font_regualar